io.github.teamclouday.android-mic.json

```diff
@@ -0,0 +1,84 @@
+{
+  "id": "io.github.teamclouday.android-mic",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "24.08",
+  "base": "com.system76.Cosmic.BaseApp",
+  "base-version": "stable",
+  "sdk": "org.freedesktop.Sdk",
+  "sdk-extensions": ["org.freedesktop.Sdk.Extension.rust-stable"],
+  "command": "android-mic",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--device=dri",
+    "--share=network",
+    "--talk-name=com.system76.CosmicSettingsDaemon",
+    "--filesystem=xdg-config/cosmic:ro",
+    "--device=usb",
```

Comment:
This will not work on older flatpak versions

Reply:
> This will not work on older flatpak versions

you mean only the ""--device=usb"," line ?

Reply:
yes